addons/point_of_sale/views/pos_config_view.xml

```diff
@@ -129,6 +117,21 @@
                                 </div>
                             </div>
                         </div>
+                        <div class="col-12 col-lg-6 o_setting_box" attrs="{'invisible': [('limit_categories', '=', False)]}">
```

Comment:
remove the invisible attribute. start category is also active if no category limit is set. making this invisible can result in start category set and no possibility to remove it. Also the user should be able to set the start category if they don't want to limit the categories available in the PoS.